procedures/release-duty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -7,24 +7,13 @@ Intro
 This manual describes how to set up a new ESR branch. The same process
 can be applied for any branch set up, with slight modifications.
 
-Example tracking bug: `Bug 1644817 - [meta] Support ESR78
-<https://bugzilla.mozilla.org/show_bug.cgi?id=esr78>`__
+Example tracking bug: `Bug 1717527 - tracking bug for build and release
+of Firefox
+91.0esr <https://bugzilla.mozilla.org/show_bug.cgi?id=1717527>`__
```

Comment:
This is neat, thanks for adjusting this, the lattest ESR bug should be always on top 👍